BLSQ/iaso
# Polio 849 update workflow fields

Some budget workflow fields have been added, deleted or renamed. They are hard coded in both front and back end for the budget tab, so they have to be updated

Related JIRA tickets : POLIO-849

## Self proofreading checklist


## Changes

- Backend: renamed/deleted fields in `Campaign`model, added migrations
- Fronte-end: updated constants used in Budget tab, adapted translations

## How to test
-  You need the latest config from the [Drive](https://drive.google.com/drive/u/0/folders/1ua4yzyFOGayweQVNKP0hyBNDKhQrPlx1). The file name is `budget_workflow_2023-02-20`
- You need to use this config in your local DB: Django admin > polio > workflowmodel
- The go tiobudget tab and check that all fields function correctly


## Print screen / video

![Screenshot 2023-02-20 at 17 08 37](https://user-images.githubusercontent.com/38907762/220155188-2db3b2ae-08ba-4a97-b697-4fa6a93f19ec.png)

## Notes

⚠️ Be careful before deploying this on `campaigns-staging`: make sure the tickets related to the workflow update have been tested and approved (POLIO-817, POLIO-748, POLIO-794), and that the config has been updated on that environment ⚠️

DO NOT MERGE UNTIL WE HAVE CONFIRMATION THAT THE CLIENT IS DONE INPUTTING DATA IN THE BUDGET TAB
